examples/dolly-pythia-fine-tuned/README.md

```diff
@@ -0,0 +1,12 @@
+# Dolly vs. Pythia Comparison
+
+The notebooks in this directory are used to:
+
+- Generate responses from prompts from the Alpaca dataset, an instruction-following dataset, using two generative LLM models, Dolly and Pythia. Pythia is a generative LLM from [Eleuther AI](https://www.eleuther.ai/). Dolly is a Pythia base model that has been fine-tuned on an instruction-following dataset (also named Dolly).
+- Extract and visualize embeddings from each model during response generation.
+- Evaluate each model using a more powerful LLM, in this case, GPT-4.
+- Visualize your data in Phoenix.
+
+The data used in `dolly_vs_pythia.ipynb` was created from the `create_data_with_dolly_and_pythia.ipynb` notebook. You can either
```

Comment:
not sure if you need to site the data

@misc{alpaca,
  author = {Rohan Taori and Ishaan Gulrajani and Tianyi Zhang and Yann Dubois and Xuechen Li and Carlos Guestrin and Percy Liang and Tatsunori B. Hashimoto },
  title = {Stanford Alpaca: An Instruction-following LLaMA model},
  year = {2023},
  publisher = {GitHub},
  journal = {GitHub repository},
  howpublished = {\url{https://github.com/tatsu-lab/stanford_alpaca}},
}

Reply:
https://huggingface.co/datasets/tatsu-lab/alpaca